modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -188,6 +190,21 @@ def map_yes_no_to_boolean(key, value)
             nil
           end
         end
+
+        def map_bgs_tracked_items
+          bgs_response = bgs_service.tracked_items.find_tracked_items(params[:id])
```

Comment:
We can't use `params[:id]` here.
The `/claims/:id` endpoint can accept a Lighthouse Id or a BGS id.
We'll want to use the value of `upstream_id` in the `build_claim_structure()` signature.